docs/getting_started/index.rst

```diff
@@ -179,6 +179,37 @@ Powershell Prompt.  Regardless of OS and shell, the following steps are the same
    output. You can report problems on the |github-issues|
    (Please try to be specific about the command and the offending output.)
 
+**Install IDAES using Conda**
+
+As an alternative to the ``pip install`` method described above, IDAES can also be installed using the Conda package manager.
+
+1. Create a new Conda environment with a name of your choice (in this example, ``my-idaes-env``)::
+
+    conda create --yes --name my-idaes-env python=3.8
+
+..
+
+    .. note:: The ``--yes`` optional flag can be used to perform the installation without having it pausing to ask for confirmation.
+
+2. Activate the ``my-idaes-env`` Conda environment::
+
+    conda activate my-idaes-env
+
+..
+
+    .. note:: This step is needed when starting a new session or a new console tab/window.
+
+3. Install the IDAES Conda package using the ``conda install`` subcommand::
+
+    conda install --yes -c IDAES-PSE -c conda-forge idaes-pse
```

Comment:
Do you need both the IDAES-PSE and conda-forge channels?  Is there a reason we are distributing through a channel separate from conda-forge?

Reply:
That's an excellent point. At the moment (i.e. the informal "alpha/beta" phase for the Conda package) the IDAES-PSE channel is needed since we're building and uploading the packages locally, as part of the release process. The next step would be to set up the conda-forge build configuration, at which point only the `-c conda-forge` will be needed (I guess the IDAES-PSE channel serves as a "training wheels" packaging solution, if you allow me the metaphor)